omas/omas_plot.py

```diff
@@ -1040,6 +1040,52 @@ def equilibrium_summary(ods, time_index=None, time=None, fig=None, ggd_points_tr
 
     return {'ax': axs}
 
+@add_to__ODS__
+def core_profiles_currents_summary(ods, time_index=None, time=None, fig=None, **kw):
```

Comment:
This function plots all in one axis. So it would be more general to not take `fig` as an input argument, but rather take `ax`. This would allow this function to be used in someone's figure (maybe with different subplots). See for example the `core_profiles_pressures(   )` function.